plugins/inventory/now.py

```diff
@@ -418,12 +418,21 @@ def _merge_instance_config(self, instance_config, instance_env):
         return instance
 
     def _get_instance_from_env(self):
+        client_secret = os.getenv("SN_SECRET_ID")
+        if client_secret:
```

Comment:
```suggestion
        if client_secret is not None:
```
Setting `SN_SECRET_ID` to an empty string would bypass this deprecation warning.